tests/plugins/colorbutton/colorbutton.js

```diff
@@ -164,6 +164,34 @@
 
 				wait();
 			} );
-		}
+		},
+
+		// (#1084)
+		'test changing text color to automatic': testAutomaticColor(),
+
+		// (#1084)
+		'test changing background color to automatic': testAutomaticColor( true )
 	} );
+
+	function testAutomaticColor( isBackgroundColor ) {
+		return function() {
+			var editor = this.editor,
+				bot = this.editorBot,
+				colorBtn = editor.ui.get( isBackgroundColor ? 'BGColor' : 'TextColor' );
+
+			bot.setHtmlWithSelection( 'Foo [<span style="' + ( isBackgroundColor ? 'background-' : '' ) + 'color:red">bar</span>]' );
+
+			editor.once( 'panelShow', function() {
+				resume( function() {
```

Comment:
Is this what you [meant](https://github.com/ckeditor/ckeditor-dev/pull/2210#discussion_r202962326) @mlewand ?

Reply:
Correct